draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1659,105 +1661,105 @@ The following shows the user information used in this example, included a claim
 }
 ```
 
-Hiding just this claim, the following SD-JWT payload would result:
+Hiding just this claim, the SD-JWT payload shown in the following would result. Note that the claims are sorted alphabetically as described in (#blinding-claim-names).
 
 {#example-simple_structured_some_blinded-sd_jwt_payload}
 ```json
 {
-  "iss": "https://example.com/issuer",
   "cnf": {
     "jwk": {
+      "e": "AQAB",
       "kty": "RSA",
       "n": "pm4bOHBg-oYhAyPWzR56AWX3rUIXp11_ICDkGgS6W3ZWLts-hzwI3x656
         59kg4hVo9dbGoCJE3ZGF_eaetE30UhBUEgpGwrDrQiJ9zqprmcFfr3qvvkGjt
         th8Zgl1eM2bJcOwE7PCBHWTKWYs152R7g6Jg2OVph-a8rq-q79MhKG5QoW_mT
         z10QT_6H4c7PjWG1fjh8hpWNnbP_pv6d1zSwZfc5fl6yVRL0DV0V3lGHKe2Wq
         f_eNGjBrBLVklDTk8-stX_MWLcR-EGmXAOv0UBWitS_dXJKJu-vXJyw14nHSG
-        uxTIK2hx1pttMft9CsvqimXKeDTU14qQL1eE7ihcw",
-      "e": "AQAB"
+        uxTIK2hx1pttMft9CsvqimXKeDTU14qQL1eE7ihcw"
     }
   },
-  "iat": 1516239022,
   "exp": 1516247022,
+  "iat": 1516239022,
+  "iss": "https://example.com/issuer",
   "sd_digest_derivation_alg": "sha-256",
   "sd_digests": {
-    "sub": "OMdwkk2HPuiInPypWUWMxot1Y2tStGsLuIcDMjKdXMU",
-    "given_name": "AfKKH4a0IZki8MFDythFaFS_Xqzn-wRvAMfiy_VjYpE",
-    "family_name": "eUmXmry32JiK_76xMasagkAQQsmSVdW57Ajk18riSF0",
-    "email": "-Rcr4fDyjwlM_itcMxoQZCE1QAEwyLJcibEpH114KiE",
-    "phone_number": "Jv2nw0C1wP5ASutYNAxrWEnaDRIpiF0eTUAkUOp8F6Y",
-    "5a2W0_NrlEZzfqmk_7Pq-w":
-      "gc8VzGTImYRXzP6j7q5RomXt2C_wtsOJ3hAHJdTuEIY",
-    "other_secret_club_membership_no":
-      "IirAwgN-MubteYvJ4fmq04p9PnpRTf7hqg0dzSWRboA",
+    "HS4QoeE9ty-I8BZTEupSzw":
```

Comment:
this claim should be after "given_name" if it is included in the alphabetical reordering too. If alphaberical reordering does not apply to blinded claim names, it should be clarified

Reply:
This is fine, as the uppercase letters in this case are ordered after the lower case ones. The precise method of alphabetical order does not matter for the purpose of hiding the original order.

Reply:
Added clarifications!